bundles/customer-password-updated-at-connector/src/FondOfKudu/Client/CustomerPasswordUpdatedAtConnector/CustomerPasswordUpdatedAtConnectorDependencyProvider.php

```diff
@@ -31,9 +34,11 @@ public function provideServiceLayerDependencies(Container $container): Container
      */
     protected function addZedRequestClient(Container $container)
     {
-        $container->set(static::SERVICE_ZED, function (Container $container) {
-            return $container->getLocator()->zedRequest()->client();
-        });
+        $container[static::CLIENT_ZED_REQUEST] = static function (Container $container): CustomerPasswordUpdatedAtConnectorToZedRequestClientInterface {
+            return new CustomerPasswordUpdatedAtConnectorToZedRequestClientBridge(
+                $container->getLocator()->zedRequest()->client(),
+            );
```

Comment:
I think it's not necessary to bridge the zed request client, you can directly pass that into your stub file.

Reply:
there's always room for debate about the bridges :) 

In the 'old' modules of Spryker, this hasn't been implemented yet, only in the newer ones. I picked it up from there :)